README.md

```diff
@@ -4,17 +4,59 @@
 [![Discord](https://img.shields.io/badge/chat-on%20disord-green.svg?logo=discord)](https://discord.gg/5E9zrhguTy)
 
 
-> 'JS Make' - parallel task runner for the frontend ecosystem with a JS extension system.
+Chomp is frontend task runner with advance featured focused on _ease-of-use_ and _not getting in the way_!
+
+## Why is Chomp easy to use?
+
+Chomps sets itself up with a single commmand!
+
+```bash
+chomp --init --import-scripts
+```
+
+Now you can run your npm scripts using Chomp!
+
+- _Where you ran `npm run <task>` before, now you can run `chomp <task>`!_
+
+The only difference is, with Chomp—it's faster. And, with a few more tweaks, you can enable smart caching, parallelism, and more!
+
+## What features does Chomp provide?
+
+Chomp is an advanced task runner. It provides features similar to [turbo](https://turbo.build/repo) and [nx](https://nx.dev/) but focuses on ease of use, *not monorepos.
```

Comment:
If removing the `JS Make` comment, I think it's worth adding an extra line here:

```
Chomp is also closer to older build techniques like Make files, but with added features and extensions for frontend projects.
```

or something like that.